pkg/codegen/hcl2/binder_schema_test.go

```diff
@@ -1,18 +1,22 @@
-package hcl2
+package hcl2_test
```

Comment:
Some rejigging required by `import cycle not allowed in test` linting error

Reply:
Ah, did this cause a cycle between `codegen/hcl2` and `codegen/internal/test`?

Reply:
indeed